lib/enums/query/props/server_properties.py

```diff
@@ -27,7 +27,10 @@ def _get_aliases():
         A method that returns all valid string alias mappings
         """
         return {
-            ServerProperties.HYPERVISOR_NAME: ["hv_name", "hypervisor_name"],
+            ServerProperties.HYPERVISOR_NAME: [
+                "hv_name",
+                "hypervisor_name",
+            ],  # Changed from Host ID as hosts aren't the same as hypervisors
```

Comment:
Nitpick - "we're using hypervisor name as the shared-common property because the hypervisor id was inconsistent between servers and hypervisors"